client/src/components/main/App/DemoDropdown.tsx

```diff
@@ -0,0 +1,75 @@
+import { FormControl, Select, MenuItem, makeStyles } from "@material-ui/core";
+import React from "react";
+import {
+  assayedDemoList,
+  categoricalDemoList,
+  getDemoObject,
+} from "../../../services/main";
+import {
+  ClientAssayedFusion,
+  ClientCategoricalFusion,
+} from "../../../services/ResponseModels";
+
+const useStyles = makeStyles(() => ({
+  demoMenu: {
+    width: "200px",
+    height: "40px",
+    alignItems: "center",
+    backgroundColor: "white",
+    marginTop: "5px",
+    marginRight: "5px",
+  },
+}));
+
+interface DemoDropdownProps {
+  handleClear: () => void;
+  handleDemo: (fusion: ClientAssayedFusion | ClientCategoricalFusion) => void;
+}
+
+export default function DemoDropdown(
+  props: DemoDropdownProps
+): React.ReactElement {
+  const classes = useStyles();
+  const { handleClear, handleDemo } = props;
+  const [selectedDemo, setSelectedDemo] = React.useState("");
+  const demoIsSelected = selectedDemo && selectedDemo !== "none";
+
+  const path = window.location.pathname;
+  let demoData = assayedDemoList;
+  if (path.includes("/categorical-fusion")) {
+    demoData = categoricalDemoList;
+  }
+
+  const handleChange = (event) => {
+    const value = event.target.value;
+    setSelectedDemo(value as string);
```

Comment:
if you select a new demo item, but then hit "cancel", the selected demo dropdown will still show the newer selection. This might get wonky but I think you have to pass down the dialog callback/open states from `App.tsx` down to this component so that you can handle that stuff before finalizing the changed demo state?

(actually maybe it's enough to tweak handleClear a bit? I'm confusing myself)

Reply:
oops. I'll fix that!

Reply:
fixed in latest commit!